attractions/checkbox/checkbox.svelte

```diff
@@ -70,8 +71,8 @@
     {disabled}
     on:change={e =>
       dispatch('change', {
-        value: e.target.value,
-        checked: e.target.checked,
+        value: e.currentTarget.value,
```

Comment:
Why this? As far as I know, `currentTarget` is a more problematic field than `target` in regards to browser compatibility

Reply:
Actually, according to caniuse.com, [`currentTarget`](https://caniuse.com/mdn-api_event_currenttarget) is supported even more than [`target`](https://caniuse.com/mdn-api_event_target), with the only difference being in IE 6-8 (so who gives a damn?).
As for why, there is a minor difference between both. `target` is the element where the event happened, while `currentTarget` is the element to which the listener is attached. They are the same in our case, but Svelte seems to augment the type for `currentTarget` so that it is not nullable and has the correct element type.

Reply:
Huh. Well, alright, as long as nothing breaks (though I'm kind of scared that it will, maybe not for this component but for another)